python/importer/remove_old_data.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+import logging
+from common.utils import get_conn_params
+
+def main():
+    logger = logging.getLogger('importer')
+
+    conn = psycopg2.connect(**get_conn_params())
+    try:
+        with conn:
+            with conn.cursor() as cur:
+                logger.info("Removing data.")
```

Comment:
Logging the "older than" amount would be useful, e.g. `Removing data older than 2 weeks`.

By the way, this "2 weeks" is something that we most probably would like to parameterize, so we can flexibly adjust the data retention period in our system. But this is something non-MVP we can do later when needed 😄